include/etl/frame_check_sequence.h

```diff
@@ -107,7 +107,7 @@ namespace etl
     //*************************************************************************
     /// Default constructor.
     //*************************************************************************
-    frame_check_sequence()
+    ETL_CONSTEXPR14 frame_check_sequence() : frame_check{}
```

Comment:
The CI fails for C++03 syntax check.
You will have to change `frame_check{}` to `frame_check()` for C++03 compatibility.